lib/Mojolicious/Renderer.pm

```diff
@@ -214,6 +220,15 @@ sub _next {
   return join '/', 'layouts', $layout;
 }
 
+sub _render_content {
```

Comment:
Is this method just dead code? Seems like it's never called.

Reply:
Right, it’s no longer used.